### ISSUE TITLE
Provide page titles, make them brief

### ISSUE THREAD

**Issue by messman** [htmllint/htmllint#71]
From [IITAA Standards, 1.3](http://www.dhs.state.il.us/IITAA/IITAAWebImplementationGuidelines.html):
Every web page should have a title. The title should indicate both the name of the site and the topic of the page and should be unique within the site whenever possible.
Note: Titles should usually be 60 or fewer characters in length.
